### PR TITLE
style: fix header and sidebar on scroll

### PR DESCRIPTION
**Pull request by vanshika703**

### CODE REVIEW

File: src/components/ButtonBar.js [hunk 1]
```diff
@@ -1,13 +1,27 @@
-
-
 const ButtonBar = () => {
-    const categories = ["All", "Mixes", "Music", "Comedy", "ReactJS", "Standup", "Cricket", "Mixes", "Mixes", "WebDev", "Travel", "Standup",]
+  const categories = [
```
Comment (harshgoel05) [inline src/components/ButtonBar.js]: Move it to a constants folder new file

File: src/components/ButtonBar.js [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
+    "WebDev",
+    "Travel",
+  ];
   return (
-    <div>
-        {
-            categories.map((category) => <button key={category} className="py-1 px-4 m-3 bg-stone-100 rounded-lg">{category}</button>)
-        }
-      
+    <div className="sticky top-0 w-full bg-white z-10">
```
Comment (harshgoel05) [inline src/components/ButtonBar.js:6]: just like top-0  we can use something [inset](https://tailwindcss.com/docs/top-right-bottom-left)

File: src/components/Header.js [hunk 3]
```diff
@@ -50,14 +50,14 @@ const Header = () => {
         />
         <img
           alt="logo"
-          className="h-7 m-1"
+          className="h-6 m-1"
           src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/b8/YouTube_Logo_2017.svg/2560px-YouTube_Logo_2017.svg.png"
```
Comment (harshgoel05) [inline src/components/Header.js:56]: All SRC in constant folder and give proper alt - what is `logo` which logo?

File: src/components/Header.js [hunk 4]
```diff
@@ -86,16 +86,37 @@ const Header = () => {
         )}
       </div>
 
-      <div className="flex col-span-1">
+      <div className="flex col-span-2">
+        <svg
```
Comment (harshgoel05) [inline src/components/Header.js]: Avoid putting svg directly, make a svg file and import here

File: src/components/Header.js [hunk 5]
```diff
@@ -86,16 +86,37 @@ const Header = () => {
...
-          className="h-7 m-1"
-          src="https://cdn4.iconfinder.com/data/icons/ionicons/512/icon-ios7-bell-512.png"
-        ></img>
-        <img
-          alt="logo"
-          className="h-7 m-1"
-          src="https://www.iconpacks.net/icons/2/free-user-icon-3296-thumb.png"
+          alt="Avatar"
+          className="h-6 w-6 rounded-full mx-2"
+          src="https://yt3.ggpht.com/yti/AHXOFjVTgdjtNwEngLpaMQ5NuEqhW_G-jbbVLvM17UtF=s88-c-k-c0x00ffffff-no-rj-mo"
```
Comment (harshgoel05) [inline src/components/Header.js:97]: Same if this is contstant src

File: src/components/Sidebar.js [hunk 6]
```diff
@@ -1,24 +1,73 @@
...
+import { ReactComponent as Library } from "../utils/img/library.svg";
+import { ReactComponent as Shorts } from "../utils/img/shorts.svg";
+import { ReactComponent as Subscriptions } from "../utils/img/subscriptions.svg";
+import { ReactComponent as YourVideos } from "../utils/img/yourvideos.svg";
+import { ReactComponent as History } from "../utils/img/history.svg";
+import { ReactComponent as LikedVideos } from "../utils/img/likedvideos.svg";
+import { ReactComponent as WatchLater } from "../utils/img/watchlater.svg";
+
+const sidebarItems = [
+  {
```
Comment (harshgoel05) [inline src/components/Sidebar.js]: Move to constants folder

File: src/components/Sidebar.js [hunk 7]
```diff
@@ -1,24 +1,73 @@
...
 
   if (!isMenuOpen) return null;
 
   return (
-    <div className="col-span-1 text-left m-2 p-2">
+    <div className="flex flex-col w-2/12 text-left m-2 p-2">
       <ul className="my-4">
         {sidebarItems &&
           sidebarItems.map((sidebarItem, index) => (
             <li key={index} className="p-2">
```
Comment (harshgoel05) [inline src/components/Sidebar.js]: change key

File: src/components/VideoCard.js [hunk 8]
```diff
@@ -4,11 +4,12 @@ const VideoCard = ({ info }) => {
   
 
   const { snippet, statistics } = info;
+  console.log(snippet)
```
Comment (harshgoel05) [inline src/components/VideoCard.js]: remove log

File: src/components/VideoCard.js [hunk 9]
```diff
@@ -4,11 +4,12 @@ const VideoCard = ({ info }) => {
...
   return (
-    <div className="w-96 m-3 p-3">
+    <div className="m-2 p-2">
       <img
         alt="thumbnail"
-        className="rounded-lg"
+        className="rounded-lg w-[360px]"
```
Comment (harshgoel05) [inline src/components/VideoCard.js:11]: define 360px in tailwind config

File: src/App.css [hunk 10]
```diff
@@ -1,3 +1,15 @@
+
 @tailwind base;
```
Comment (harshgoel05) [inline src/App.css]: Move all these to index.css and delete app.css

**Review by harshgoel05 — CHANGES_REQUESTED** [review]

File: src/components/ButtonBar.js (hunk 2 above)
Comment (vanshika703) [reply src/components/ButtonBar.js:6]: Okayy

File: src/components/VideoCard.js (hunk 9 above)
Comment (vanshika703) [reply src/components/VideoCard.js:11]: I have not defined it separately in the config file as it is used only once.

**Pull request merged**